common/app/views/fragments/stylesheetsUse.scala.html

```diff
@@ -52,15 +52,6 @@
 
             // GO!
             useCss();
-
```

Comment:
good spot - this probably should have been associated with a switch so it did get remembered